examples/README.md

```diff
@@ -414,6 +414,51 @@ exact_match = 86.91
 This fine-tuneds model is available as a checkpoint under the reference
 `bert-large-uncased-whole-word-masking-finetuned-squad`.
 
+#### Fine-tuning RoBERTa on SQuAD
+
+This is an example using 4-GPUs distributed training to fine-tune RoBERTa-large model on the SQuAD v2.0 dataset:
```

Comment:
Can you add the model of the GPU so it will be easy to tune the per_gpu_eval_batch_size relatively to the memsize of the GPU used in the example